daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_expressions/expressions.tdml

```diff
@@ -2010,12 +2010,13 @@
     model="expressions-Embedded.dfdl.xsd" description="">
 
     <tdml:document><![CDATA[321blastoff]]></tdml:document>
-    <tdml:errors>
-      <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>property 'initiator'</tdml:error>
-      <tdml:error>cannot start or end with the string</tdml:error>
-      <tdml:error>U+0020</tdml:error>
-    </tdml:errors>
+    <tdml:infoset>
+      <tdml:dfdlInfoset>
+        <expr_space5>
+          <value>21blastoff</value>
```

Comment:
This output is correct, as the newlines in the initiator are whitespace, and therefore delimeters. However, just looking at the test, it looks wrong for the "21" to not be part of the prefix. Maybe have the initiator be
```
"{'
321
blast
' }"
```

That way the example is intuitive, and you still demonstrate that the newline is being interperated as a delimeter.